timescaledb/how-to-guides/connecting/dbeaver.md

```diff
@@ -1,15 +1,118 @@
 ---
 title: DBeaver
 excerpt: Connect to your TimescaleDB database with DBeaver
+product: mst, cloud, timescaledb
 keywords: [connect]
 ---
 
-# DBeaver
-DBeaver is a free database tool that is available for Microsoft Windows, Apple
-macOS, or various Linux flavors. You can also use it as a plugin to the Eclipse
-IDE platform.
+# Connect to TimescaleDB using DBeaver
 
-For more information about DBeaver, including download and installation
-instructions, see [dbeaver.io][dbeaver].
+[DBeaver][dbeaver] is a free and open source database tool that is
+available for Microsoft Windows, Apple macOS, and many Linux versions. DBeaver
+provides a powerful SQL editor, administration features, ability to migrate data
+and schema, and the ability to monitor database connection sessions. You can
+connect to TimescaleDB hosted on your local machine or on a remote server. You
+can also connect to Timescale Cloud, and Managed Service for TimescaleDB
+services.
+
+## Before you begin
+
+*   Download and install [DBeaver][dbeaver-downloads].
+
+### Connecting to on-premise TimescaleDB using DBeaver
+
+<procedure>
+
+1.  Start `DBeaver`.
+1.  In the `Database` menu, click `New Database Connection`.
+1.  In the `Connect to a database` window, search for `TimescaleDB`.
+1.  Select `TimescaleDB` and click `Next` to continue.
+1.  In the `Main` tab, type the details for your connection, including the
+ `Host`, `Port`,`Database`, `Username` and `Password`. For more information
+ about these settings, see [the connection settings section][connection-settings].
+1.  Click the `Test Connection` button to check if the connection is successful.
+1.  Click the `Finish` button to connect to the database server. The server is
+    listed in the `Database Navigator` pane.
+
+    <img class="maincontent__illustration"src="https://s3.amazonaws.com/assets.timescale.com/docs/images/on_premise_dbeaver.png"alt="Connecting to TimescaleDB using DBeaver"/>
+
+</procedure>
+
+### Connecting to Timescale Cloud with DBeaver
+
+<procedure>
+
+1.  Sign in to the [Timescale Cloud portal][tsc-portal].
+1.  In the `Services` tab, find the service you want to connect to, and check
+    it is marked as `Running`.
+1.  Click the name of the service you want to connect to, and take a note
+     of the `Host`, `Port`, `Database name`, and `Username`.
+1.  Navigate to the `Operations` tab, and click `Reset password`. You can choose
```

Comment:
should it be marked as an optional step? Eg.: In case you don't know the password of the instance you want to connect to, you can always reset the password.